README.md

```diff
@@ -1 +1,42 @@
-# Hello_Microverse_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+This is my first Microverse Project
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Getting Started
```

Comment:
- [ ] You can edit the content of this section by adding steps necessary to setup this project. e.g `git clone... ` This will immediately help anyone trying to get your project running on their local machine